test/trafos/backends/pyfftw_bindings_test.py

```diff
@@ -0,0 +1,422 @@
+# Copyright 2014, 2015 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+
+from future import standard_library
+standard_library.install_aliases()
+
+from itertools import product
+from math import pi
+import numpy as np
+import pytest
+
+from odl.trafos.backends import pyfftw_call, PYFFTW_AVAILABLE
+from odl.util import (all_almost_equal, all_equal, never_skip,
+                      is_real_dtype, conj_exponent, TYPE_MAP_R2C)
+
+
+pytestmark = pytest.mark.skipif(not PYFFTW_AVAILABLE,
+                                reason='`pyfftw` backend not available')
+
+
+def _random_array(shape, dtype):
+    if is_real_dtype(dtype):
+        return np.random.rand(*shape).astype(dtype)
+    else:
+        return (np.random.rand(*shape).astype(dtype) +
+                1j * np.random.rand(*shape).astype(dtype))
+
+
+# ---- pyfftw_call ---- #
+
+
+def _params_from_dtype(dt):
```

Comment:
Why would it do that? I'm quite sure it wouldnt since this is not a fixture nor a test, no?